### PR TITLE
fix: update help.aiven.io link

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes
Fix the link from a help.aiven.io article to point to the Karapace docs.

## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [X] I applied the [style guide](../styleguide.md).
- [X] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
 
 As of version 3.0, Aiven for Apache Kafka no longer supports Confluent
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md:46]:
```suggestion
Starting with version 3.0, Aiven for Apache Kafka no longer supports Confluent
```

File: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
...
 Schema Registry. For more information, read [the article describing the
-replacement, Karapace](https://help.aiven.io/en/articles/5651983)
+replacement, Karapace](https://aiven.io/docs/products/kafka/karapace).
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md:48]:
```suggestion
replacement, Karapace](/docs/products/kafka/karapace).
```

File: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md:46]: This was probably added when it was only Apache Kafka version 3.0; we now support version 3.6.

File: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-source.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
 
 As of version 3.0, Aiven for Apache Kafka no longer supports Confluent
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-source.md:47]:
```suggestion
Starting with version 3.0, Aiven for Apache Kafka no longer supports Confluent
```

File: docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-source.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
...
 Schema Registry. For more information, read [the article describing the
-replacement, Karapace](https://help.aiven.io/en/articles/5651983)
+replacement, Karapace](https://aiven.io/docs/products/kafka/karapace).
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-source.md:49]:
```suggestion
replacement, Karapace](/docs/products/kafka/karapace).
```

**Review by ArthurFlageul — APPROVED** [review]

**Pull request merged**